src/reducers/url-state.js

```diff
@@ -17,6 +17,14 @@ import type {
 import type { UrlState, Reducer } from '../types/state';
 import type { TabSlug } from '../app-logic/tabs-handling';
 
+/*
```

Comment:
I think we need a newline between this comment and the first reducer. I wish we had a better style for module-level comments. I get them confused with function comments.